app/operators/clinrev_pdf_operator.py

```diff
@@ -0,0 +1,286 @@
+# Clinical Review PDF generator
+import monai.deploy.core as md
+import pydicom
+from monai.deploy.core import DataPath, ExecutionContext, InputContext, IOType, Operator, OutputContext
+import logging
+import nilearn
+import pathlib
+import os.path
+from os import listdir
+from os.path import isfile, join
+import os
+import os.path
+import monai.deploy.core as md
+from monai.deploy.core import DataPath, ExecutionContext, InputContext, IOType, Operator, OutputContext
+from monai.deploy.core.domain.dicom_series_selection import StudySelectedSeries
+from monai.deploy.core.domain import DICOMStudy
+from monai.deploy.core.domain.dicom_sop_instance import DICOMSOPInstance
+from pathlib import Path
+import SimpleITK as sitk
+
+import matplotlib.pyplot as plt
+import numpy as np
+import nibabel as nib
+
+from typing import TYPE_CHECKING, Dict, List, Optional, Sequence
+import reportlab.platypus as pl # import Table, TableStyle, Image
+from reportlab.platypus import Table, TableStyle, Image
+from reportlab.lib.pagesizes import letter, A4
+from reportlab.platypus import SimpleDocTemplate, Paragraph, Spacer, KeepInFrame
+from reportlab.lib.styles import getSampleStyleSheet, ParagraphStyle
+from reportlab.lib.units import inch, cm
+from reportlab.lib import utils
+
+from reportlab.lib import colors
+from reportlab.lib.colors import white, black
+from reportlab.lib.fonts import tt2ps
+from reportlab.rl_config import canvas_basefontname as _baseFontName
+_baseFontNameB = tt2ps(_baseFontName, 1, 0)
+_baseFontNameI = tt2ps(_baseFontName, 0, 1)
+_baseFontNameBI = tt2ps(_baseFontName, 1, 1)
+
+
+@md.input("input_files", DataPath, IOType.DISK)
+@md.output("pdf_file", DataPath, IOType.DISK)
+@md.output("study_selected_series", List[StudySelectedSeries], IOType.IN_MEMORY)
+@md.env(pip_packages=["pydicom >= 2.3.0", "highdicom >= 0.18.2"])
+class ClinicalReviewPDFGenerator(Operator):
+
+    """Generates a PDF with Sag/Cor/Ax views for each structure"""
+
+    def compute(self, op_input: InputContext, op_output: OutputContext, context: ExecutionContext):
+
+        logging.info(f"Begin {self.compute.__name__}")
+        # get list of masks
+        input_path = op_input.get("input_files").path
+        dcm_input_path = input_path / 'dcm_input'
+        nii_seg_output_path = dcm_input_path / 'nii_seg_output'
```

Comment:
To get it working on my computer with a set of DICOM files in the `/input` directory, I had to change this line to:

`nii_seg_output_path = input_path / 'nii_seg_output'`